nx-X11/programs/Xserver/dix/window.c

```diff
@@ -2350,6 +2354,7 @@ ConfigureWindow(register WindowPtr pWin, register Mask mask, XID *vlist, ClientP
 	if (size_change && ((pWin->eventMask|wOtherEventMasks(pWin)) & ResizeRedirectMask))
 	{
 	    xEvent eventT;
+	    memset(&eventT, 0, sizeof(xEvent));
```

Comment:
This should be also added to line 2404 (not with `eventT`, but `event`.)

(Re-adding this comment so that it's not removed if you rebase the commits.)